lib/SILOptimizer/Mandatory/MoveOnlyAddressCheckerUtils.cpp

```diff
@@ -621,11 +624,11 @@ struct UseState {
 
   /// A map from an instruction that initializes memory to the description of
   /// the part of the type tree that it initializes.
-  llvm::SmallMapVector<SILInstruction *, TypeTreeLeafTypeRange, 4> initInsts;
+  llvm::SmallMapVector<SILInstruction *, SmallBitVector, 4> initInsts;
```

Comment:
@nate-chandler shouldn't this also use inst to bit map?

Reply:
Yep